dags/nocp/scripts/run_ocm_benchmark.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+export UUID=$(uuidgen | head -c8)-$AIRFLOW_CTX_TASK_ID-$(date '+%Y%m%d')
+echo "############################################"
+echo "# Benchmark UUID: ${UUID}"
+echo "############################################"
+
+env >> /tmp/environment.txt
+echo $UUID
+
+run_ocm_benchmark(){
```

Comment:
Thoughts on adding the setup steps for the orchestration host before running the benchmark? This way if we pick a new host we dont have to do any of the dirty work. Feels like it would be a pretty simple add unless I'm missing something.

Reply:
Below are the steps (documented at https://docs.google.com/document/d/1QdKyi5tY1m0ytvGjx7zK8okeAM10soQJq87m4eetaAc/edit?usp=sharing ) needed to succesfully run ocm-api-load. We already got dedicated VM "airflow-ocm-jumphost.rdu2.scalelab.redhat.com" from scale lab for CI. We can enhance this script file with below steps later if we really want to run on any host.

1. Packages for building ocm-api-load binary
i) dnf groupinstall "Development tools" -y
ii) sudo yum install git golang -y

2. In order for airflow to login to jumphost successfully, we need to add its public key to jumphost’s authorized_key
i) Clone the perf-dept where key is present “git clone -q --depth=1 --single-branch --branch master https://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@github.com/redhat-performance/perf-dept.git /tmp/perf-dept”
ii) Copy key from “perf-dept/ssh_keys/id_rsa_pbench_ec2.pub”

3) Install below packages for automation.py module in ocm-api-load
i) yum install python3-devel -y
ii) pip3 install pandas plotly requests sqlalchemy docx python-docx

4) Ocm-api-load requires aws key with admin user, so configure aws
i)sudo yum install awscli jq -y
ii)aws configure --profile=openshift-perfscale
ii)Add “export AWS_PROFILE=openshift-perfscale” in ~/.bashrc file